documentation/index.ts

```diff
@@ -0,0 +1,497 @@
+        rightsString += `Right needed *connectivity environment*: ${right.supplier}\\n`;
+      }
+      if (right.environment !== undefined) {
+        rightsString += `Right needed *monitoring environment*: ${right.environment}\\n`;
+      }
+    }
+    return `"description": "${description !== undefined ? description : ''}${right !== undefined ? ` ${rightsString}` : ''}",`;
+  }
+
+  const methodContentParts: string [] = [];
+  tags.add(tag);
+  routesToDocument.forEach((route) => {
+    methodContentParts.push(`"${route.method}": {
+    ${route.summary !== undefined ? `"summary": "${route.summary}",` : ''}
+    ${determineDescription(route.description, route.right)}
+    "responses": {
+      ${determineResponseContent(route)}
+    }
+    ${determineRequestBodyContent(route)},
+    "tags": ["${tag}"]
+  }`);
+  });
+  return methodContentParts.join(',');
+}
+
+function determinePathSpecification(
+  controllerGeneratorOptions: RouteToDocument[],
+): string {
+  const pathSpecificationParts: string[] = [];
+  const groupedRoutes: Record<string, RouteToDocument[]> = {};
+  controllerGeneratorOptions.forEach((controllerGeneratorOption) => {
+    const key = controllerGeneratorOption.path;
+    if (groupedRoutes[key] === undefined) {
+      groupedRoutes[key] = [];
+    }
+    groupedRoutes[key].push(controllerGeneratorOption);
+  });
+  Object.entries(groupedRoutes).forEach((record) => {
+    pathSpecificationParts.push(
+      `"/${record[1][0].tag}${record[0]}": {
+      ${determineMethodsContent(record[1], record[1][0].tag)}
+    }`,
+    );
+  });
+
+  return pathSpecificationParts.join(',');
+}
+
+function determineWiddershinsOptions() {
+  return {
+    codeSamples: true,
+    httpsnippet: false,
+    templateCallback(templateName: any, stage: any, data: any) { return data; },
+    theme: 'darkula',
+    search: true,
+    sample: true, // set false by --raw,
+    discovery: false,
+    includes: [],
+    shallowSchemas: false,
+    tocSummary: false,
+    headings: 2,
+    yaml: false,
+  };
+}
+
+function determineSchemaSpecifications(schemaSpecifications: (ComponentsSchema | undefined)[]) {
+  const setOfSchemas: Set<string> = new Set<string>();
+  const convertedSchemaSpecifications: string[] = [];
+  schemaSpecifications.forEach((schemaSpecification) => {
+    if (schemaSpecification !== undefined && schemaSpecification.schemas !== undefined) {
+      const schemas = schemaSpecification.schemas!;
+      const schemaNames = Object.getOwnPropertyNames(schemaSpecification.schemas);
+      schemaNames.forEach((schemaName: string) => {
+        if (!setOfSchemas.has(schemaName)) {
+          setOfSchemas.add(schemaName);
+          const schema = schemas[schemaName];
+          convertedSchemaSpecifications.push(`"${schemaName}": ${JSON.stringify(schema)}`);
+        }
+      });
+    }
+  });
+  return convertedSchemaSpecifications;
+}
+
+function createGeneralInformationMd(): string {
+  return `
+# Getting started
```

Comment:
Maybe we could move this to another file?

Reply:
fixed